ibis/backends/tests/test_client.py

```diff
@@ -295,3 +295,27 @@ def test_insert_overwrite_from_expr(
     con.insert(temp_table, obj=from_table, overwrite=True)
     assert len(temporary.execute()) == 3
     tm.assert_frame_equal(temporary.execute(), from_table.execute())
+
+
+@pytest.mark.only_on_backends(
+    SQLALCHEMY_BACKENDS, reason="run only if backend is SQLAlchemy based",
+)
+def test_list_databases(con):
+    # Every backend has its own databases
+    TEST_DATABASES = {
```

Comment:
could be a fixture at some point

Reply:
I don't see this being used, now or in the future, outside of this test. That's what I thought a constant here was a better option. But happy to make it a fixture if you think it's better, even if I think it'll make things less readable to me.